launcher/src/background.js

```diff
@@ -33,10 +34,24 @@ const tekuGasLimitConfig = new TekuGasLimitConfig(nodeConnection);
 const sshService = new SSHService();
 const { globalShortcut } = require("electron");
 const log = require("electron-log");
+const logFileBackup = new LogFileBackup();
 const stereumUpdater = new StereumUpdater(log, createWindow, isDevelopment);
 stereumUpdater.initUpdater();
 log.transports.console.level = process.env.LOG_LEVEL || "info";
 log.transports.file.level = "debug";
+log.transports.file.archiveLogFn = (file) => {
+  file = file.toString();
+  const info = path.parse(file);
+  let backupPath = info.dir + "/backups/";
+  if (!existsSync(backupPath)) {
+    mkdirSync(backupPath);
+  }
+
+  copyFileSync(file, `${backupPath}main-${Date.now()}.log`);
```

Comment:
You can just use rename just like in the example i send you then you don't need to copy and then use unlink. (if you're deleting files i'd always go with rm and not unlink)